drivers/i2c/i2c_ifx_cat1.c

```diff
@@ -0,0 +1,539 @@
+/*
+ * Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+ * an affiliate of Cypress Semiconductor Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @brief I2C driver for Infineon CAT1 MCU family.
+ *
+ * Note:
```

Comment:
looks like it is out of date note. Slave mode is supported now. Please remove this note.